config/default.js

```diff
@@ -25,6 +25,13 @@ module.exports = {
     clientSecret: "<clientSecret>",
   },
 
+  emailSubscriptionCredentials: {
+    user: "<RDS_EMAIL>",
+    pass: "<EMAIL PASS GENERATED AFTER 2FA>",
+    host: "<smtp host>",
+    port: "<number>",
```

Comment:
- are the steps to generate this written in any issue ticket? do we have to complete it before the pr is merged to develop?

Reply:
I haven't made it yet, but will create it before making these commits to main.

Reply:
> I haven't made it yet, but will create it before making these commits to main.

where is the ticket for this ?
please make one if not made already and add it in the pr description